roles/ceph-defaults/defaults/main.yml

```diff
@@ -228,6 +228,7 @@ ceph_iscsi_config_dev: true # special repo for deploying iSCSI gateways
 # ceph_custom_repo should be the URL to the repo base.
 #
 ceph_custom_repo: https://server.domain.com/ceph-custom-repo
+ceph_custom_key: https://server.domain.com/ceph-custom-repo-key.asc
```

Comment:
`ceph_custom_repo` isn't specific to Debian only so adding the new `ceph_custom_key` variable should also take care of RedHat system (probably via `rpm_key` module)

https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/tasks/installs/redhat_custom_repository.yml

Reply:
In the rpm based system, repository file are taken by ansible using get_url, which mostly already defined rpm_key inside.

On the debian side, even sources.list file repository can't define it's own key, so separate task have to added to add the key.

Reply:
the key defined in the yum repository will be imported implicitly during the first package installation on that repository.
To avoid trusting any gpg keys in that repository we should enforce it by importing the key explicitly before the repository configuration via `ceph_custom_key`.

This is what we already do in other tasks like [1] and what is recommended in the ceph documentation [2] for community repositories configuration.

[1] https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/tasks/installs/redhat_community_repository.yml#L9-L27
[2] https://docs.ceph.com/docs/master/install/get-packages/#add-keys

Reply:
done